server.js

```diff
@@ -0,0 +1,71 @@
+"use Strict";
+
+// Creates the path to get the data from reply.js
+const replies = require("./reply.js");
+
+// Define variables
+const PORT = process.env.PORT || 8080;
+
+// Import net module
+const net = require("net");
+
+// Create a TCP Server
+const server = net.createServer(socket => {
+  // set encoding tp utf8
+  socket.setEncoding("utf8");
+  // add "on" event listener
+  socket.on("data", data => {
+    // read incoming data
+    console.log(data.toString());
+
+    // parse the string
+    let index = data.slice(data.indexOf("/"), data.indexOf("HTTP") - 1);
+
+    // Empty Response Header
+    let response = "";
+    let resStatus = "";
+    let resType = "";
+    let resLength = 0;
+    let resConnection = "";
+
+    // grab the right file
+    replies.forEach(e => {
+      if (index === e.request) {
+        resStatus = e.status;
+        resType = e.type;
+        resLength = e.content.length;
+        resConnection = e.connection;
+        resContent = e.content;
+      }
+    });
```

Comment:
Based on how your reply.js is setup, using an array is a performance disadvantage since you'll have to loop through the replies before you find the right one. It would be faster to use an object where the key could be something like `hydrogen.html` and then use the URI that you get from your server data to try and find the right reply.